topics/metagenomics/tutorials/foodborne-pathogen-detection-nanopore-sequencing-data/tutorial.md

```diff
@@ -0,0 +1,1302 @@
+>
+{: .hands-on}
+
+We can now run multiple sequence alignment, build the trees for each VF and display them.
+
+> <hands-on-title> Phylogenetic Tree building </hands-on-title>
+>
+> 1. {% tool [ClustalW](toolshed.g2.bx.psu.edu/repos/devteam/clustalw/clustalw/2.1+galaxy1) %} with the following parameters:
+>    - {% icon param-collection %} *"FASTA file"*: output of **Filter sequences by ID** {% icon tool %}
+>    - *"Data type"*: `DNA nucleotide sequences`
+>    - *"Output alignment format"*: `FASTA format`
+>    - *"Output complete alignment (or specify part to output)"*: `Complete alignment`
+>
+> 2. {% tool [FASTTREE](toolshed.g2.bx.psu.edu/repos/iuc/fasttree/fasttree/2.1.10+galaxy1) %} with the following parameters:
+>    - *"Aligned sequences file (FASTA or Phylip format)"*: `fasta`
+>        - {% icon param-collection %} *"FASTA file"*: output of **ClustalW** {% icon tool %}
+>        - *"Set starting tree(s)"*: `No starting trees`
+>    - *"Protein or nucleotide alignment"*: `Nucleotide`
+>
+> 3. {% tool [Newick Display](toolshed.g2.bx.psu.edu/repos/iuc/newick_utils/newick_display/1.6+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"Newick file"*: output of **FASTTREE** {% icon tool %}
+>    - *"Branch support"*: `Hide branch support`
+>    - *"Branch length"*: `Hide branch length`
+>    - *"Image width"*: `2000`
+{: .hands-on}
+
+</div>
+
+> <question-title></question-title>
+>
+> Now let's see how your trees for the bacteria pathogen gene with accession IDs: **AAF98391**, **NP_249099** and **NP_460111** look like. To access that go to the output of Newick
+>
+>
+> 1. In which samples and contigs **AAF98391** gene is found?
+> 2. In which samples and contigs **NP_249099** gene is found?
+> 3. In which samples and contigs **NP_460111** gene is found?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. Only in `Barcode10` sample only in Contig 23
+> >
+> >    ![AAF98391_Part_of_tree](./images/AAF98391_Part_of_tree.png)
+> >
+> > 2. Only in `Barcode11` sample only in Contig 2
+> >
+> >    ![NP_249099_Part_of_tree](./images/NP_249099_Part_of_tree.png)
+> >
+> > 3. In both samples; `Barcode10` Contig 95 and `Barcode11` Contig 1
+> >
+> >    ![NP_460111_Part_of_tree](./images/NP_460111_Part_of_tree.png)
+> >
+> {: .solution}
+{: .question}
+
+# Conclusion
+
+In this tutorial, we have tried the workflow designed to detect and track pathogens in our food and drinks. Through out the full workflow we used our Nanopore sequenced datasets from Biolytix and analyzed it, found the pathogens and tracked it. This approach can be summarized with the following scheme:
+
+![Foodborne full workflow big picture](./images/FoodBorne-Workflow-updated.png "The complete picture of the workflow used in this training highlighting, not all, but the most important steps done in 5 sub-workflows explained in the training")
```

Comment:
It would be great to have useful alt text. See tutorial https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#adding-images-with-captions